pyam/figures.py

```diff
@@ -0,0 +1,70 @@
+import logging
+
+import pandas as pd
+import plotly.graph_objects as go
+from pyam.index import get_index_levels
+
+logger = logging.getLogger(__name__)
+
+
+def sankey(df, mapping):
+    """Plot sankey diagram of existing data using plotly.
+
+        Currently only for one year possible.
+        Parameters
+        ----------
+        df : pd.DataFrame
+            data to plot as wide format
+        mapping : dict
+            Assigns the source and target component of a variable
+
+            .. code-block:: python
+
+            {
+                variable: (source, target),
+                }
```

Comment:
this section has to be indented (beyond the `.. code-block::`) to be displayed correctly
```suggestion
                {
                    variable: (source, target),
                }
```